app/src/main/java/com/portfolio/tasky/views/TaskyAppCompatEditText.kt

```diff
@@ -53,17 +53,18 @@ class TaskyAppCompatEditText(context: Context, attrs: AttributeSet) : FrameLayou
         subLayout.drawableLast = drawableLast
     }
 
-    fun setError(error : Boolean){
+    fun setError(error : Boolean) {
         subLayout.error = error
     }
 
-    fun setValid(valid: Boolean){
+    fun setValid(valid: Boolean) {
         subLayout.valid = valid
     }
-    private fun isPassword(){
+    private fun isPassword() {
```

Comment:
Same feedback as before here - as soon as a function starts with `is` or `are` it's formulated as a question which you expect it to answer (by returning a boolean in this case). However this function doesn't do that so from the outside it's kinda cryptic what it does.
And to be honest, I don't really understand what it even does, since you assign something that sounds like a boolean (`isPassword`) to a text (`subLayout.password`). Could you clarify that? 👍

Reply:
True, the method name doesn't describe its purpose properly. 

The custom edittext's inputType is switched here, by providing 'isPassword' value to a data binding object of type boolean, when clicked on the eye icon. 

This data binding object switches the inputType of inner AppCompatEdittext inputType to 'TEXT_CLASS_TEXT / TEXT_CLASS_VARIATION_PASSWORD'